config/config.go

```diff
@@ -96,6 +96,10 @@ func SetupConfig(envFilePath string) error {
 		return errors.New("Trust Authority base URL needs to be provided in configuration")
 	}
 
+	if !strings.HasPrefix(configValues.TrustAuthorityBaseUrl, "https://") {
```

Comment:
What is someone changes the config file post installation, will this check still work ?